ci/azure-pipelines-dev.yml

```diff
@@ -206,6 +206,16 @@ stages:
             python ci/checks/check_only_name_tag_snippets.py
           name: NameTagSnippetChecker
 
+    - job: name_tag_snippet_referenced_checker
+      condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.DocsChanged'], true)
+      steps:
+        - task: PythonScript@0
+          inputs:
+            scriptSource: 'filePath' # 'filePath' | 'inline'. Required. Script source. Default: filePath.
+            scriptPath: 'ci/checks/check_name_tag_snippets_referenced.py' # Required when scriptSource == FilePath
+            failOnStderr: false # boolean. Fail on standard error. Default: false.
```

Comment:
Interesting! I wonder if we will need to change other scripts now. Also whether something changed with azure since the method you were using earlier used to work.

Reply:
Ah yes opened a PR for those: https://github.com/great-expectations/great_expectations/pull/8208 Can you take a look?